tests/models/bloom/test_embeddings_bloom.py

```diff
@@ -0,0 +1,399 @@
+import unittest
```

Comment:
Can probably be merged with the other modeling test file.

Reply:
fixed in e580cb5a5e848b27c40eb835bc93159b0dbfe182